### ISSUE TITLE
can function 'smoothed_pseudo_wigner_ville' be rewritten based on class WignerVilleDistribution or PseudoWignerVilleDistribution?

### ISSUE THREAD

**Issue by wangwuqi** [scikit-signal/tftb#170]
function smoothed_pseudo_wigner_ville is costs high computation compared with WignerVilleDistribution or PseudoWignerVilleDistribution, i'm wondering smoothed_pseudo_wigner_ville be rewritten based on the basic class.

**Comment by jaidevd** [comment]
Yes, @wangwuqi That is the intention - the current function is working correctly, but ultimately a placeholder. This will be added in the next release.
